forgerock-core/build.gradle

```diff
@@ -78,8 +78,8 @@ dependencies {
 
     implementation 'com.squareup.okhttp3:okhttp:4.11.0'
     implementation 'com.squareup.okhttp3:logging-interceptor:4.11.0'
-    implementation 'androidx.annotation:annotation:1.1.0'
-    implementation 'androidx.security:security-crypto:1.1.0-alpha05'
+    implementation 'androidx.annotation:annotation:1.6.0'
+    implementation 'androidx.security:security-crypto:1.1.0-alpha06'
```

Comment:
can we test the webauthn and device bind key survive between version upgrade (example 4.1.0 to 4.2.0)